sonar-plugin-api/src/main/java/org/sonar/api/ce/posttask/PostProjectAnalysisTaskTester.java

```diff
@@ -214,9 +214,17 @@ public Date getDate() {
       }
 
       @Override
+      public Optional<Analysis> getAnalysis() {
+        //FIXME
+        return null;
+      }
+
+      /*
+      @Override
       public Optional<Date> getAnalysisDate() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACommentedOutCodeLine)